test/arrays/types/ArrayTest.chpl

```diff
@@ -6,4 +6,8 @@ proc testArray(type t) {
 
 
   assert(a.size == 1);
+
+  if isunmanagedclass(t) {
```

Comment:
capitalization?

Reply:
Just caught that in my latest full test run -- I need to disable that vim hotkey...